internal/core/src/common/BitsetView.cpp

```diff
@@ -0,0 +1,68 @@
+// Licensed to the LF AI & Data foundation under one
+// or more contributor license agreements. See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership. The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <boost_ext/dynamic_bitset_ext.hpp>
+#include "common/Types.h"
+#include "common/BitsetView.h"
+#include "utils/Utils.h"
+
+namespace milvus {
+
+// negate bitset, and merge them into one
+aligned_vector<uint8_t>
+AssembleNegBitset(const BitsetSimple& bitset_simple) {
```

Comment:
This API should be AssembleBitset or BitsetAssemble.
To flip all bit in bitset, we can use boost_bitset::flip().

Reply:
This function is not been used ethier.